src/cosalib/aws.py

```diff
@@ -38,9 +38,18 @@ def aws_run_ore_replicate(build, args):
     if len(buildmeta.get('amis', [])) < 1:
         raise SystemExit(("buildmeta doesn't contain source AMIs."
                          " Run buildextend-aws --upload first"))
+
+    # Determine which region to copy from
+    if not args.source_region:
+        args.source_region = buildmeta['amis'][0]['name']
+
+    # If no region specified then autodetect the regions to replicate to.
+    # Specify --region=args.source_region here so ore knows to talk to
+    # a region that exists (i.e. it will talk to ovcloud if copying
+    # from a govclou region).
     if not args.region:
         args.region = subprocess.check_output([
-            'ore', 'aws', 'list-regions'
+            'ore', 'aws', f'--region={args.source_region}', 'list-regions'
```

Comment:
Minor: feels like

```suggestion
            'ore', 'aws', '--region', args.source_region, 'list-regions'
```

would be cleaner